File: PLATZI PROGRAMS/calculadora.py
# en este programa se hara una calculadora con las 4 operaciones basicas
# Autor: Tomas Alberto Rodriguez Peña

def calculadora(numero1, numero2, operador) -> str:
    resultado = ""
    match operador:
        case "+":
            resultado = ("La suma de los dos numeros es: " +
                         str(suma(numero1, numero2)))
        case "-":
            resultado = ("La resta de los dos numeros es: " +
                         str(resta(numero1, numero2)))
        case "*":
            resultado = ("La multiplicacion de los dos numeros es: " +
                         str(multiplicacion(numero1, numero2)))
        case "/":
            resultado = ("La division de los dos numeros es: " +
                         str(division(numero1, numero2)))
        case _:
            resultado = (
                "Elija por favor una operacion acorde a las funciones de la calculadora")
    return resultado


def suma(numero1, numero2) -> int:
    return numero1 + numero2


def resta(numero1, numero2) -> int:
    return numero1 - numero2


def multiplicacion(numero1, numero2) -> int:
    return numero1 * numero2


def division(numero1, numero2) -> float:
    return numero1 / numero2


def consola() -> None:
    num1 = int(input("Ingrese el primer numero que deseea operar: "))
    num2 = int(input("Ingrese el segundo numero que deseea operar: "))
    operator = input("Ingrese el tipo de operacion que desea realizar: ")

    print(calculadora(num1, num2, operator))


consola()

File: PLATZI PROGRAMS/calculadora.js
// en este programa se hara una calculadora con las 4 operaciones basicas
// Autor: Tomas Alberto Rodriguez Peña

function calculadora(numero1, numero2, operador) {
    let resultado;
    switch (operador) {
        case "+":
            resultado = suma(numero1, numero2);
            break;
        case "-":
            resultado = resta(numero1, numero2);
            break;
        case "*":
            resultado = multiplicacion(numero1, numero2);
            break;
        case "/":
            resultado = division(numero1, numero2);
            break;
        default:
            resultado = "Elija una opcion correcta"
            break;
    }

    return resultado;
}

function suma(numero1, numero2) {
    return numero1 + numero2;
}

function resta(numero1, numero2) {
    return numero1 - numero2
}

function multiplicacion(numero1, numero2) {
    return numero1 * numero2
}

function division(numero1, numero2) {
    return numero1 / numero2
}
function consola() {
    let num1 = Number(prompt("Ingrese el primer numero que deseea operar"));
    let num2 = Number(prompt("Ingrese el segundo numero que deseea operar"));
    let operator = prompt("Ingrese el tipo de operacion que desea realizar");

    console.log("El resultado de su operacion es: " + calculadora(num1, num2, operator));
}

consola()